data/jobs.txt

```diff
@@ -2292,6 +2366,119 @@ mission "Large Rush Delivery [3]"
 		payment 42000
 		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
 
+
+
+mission "Prisoners [0]"
```

Comment:
I would relocate these files (or perhaps instead, the `Large Bulk/Rush Delivery` missions) so that all the passenger missions are co-located.

Right now:
 - 25 Escort missions
 - 3 (5 with yours) Bounty Hunting missions
 - 3 Pirate Attack
 - 5 Cargo Delivery
 - 3 Bulk Delivery
 - 4 Rush Delivery
 - 5 Passenger transport
 - 4 industry worker transport
 - 6 tourist transport
 - 4 family transport
 - 3 scab transport
 - 2 colonists
 - 3 Large Bulk Delivery
 - 4 Large Rush Delivery
 - (5 Prisoner transport)
 - Recycle garbage
 - 10 Bounty Hunting (marauder)